qa/integration-tests/src/test/java/io/zeebe/broker/it/network/Ipv6IntegrationTest.java

```diff
@@ -0,0 +1,138 @@
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.broker.it.network;
+
+import com.github.dockerjava.api.model.Network.Ipam;
+import com.github.dockerjava.api.model.Network.Ipam.Config;
+import io.zeebe.client.ZeebeClient;
+import io.zeebe.client.ZeebeClientBuilder;
+import io.zeebe.client.api.response.Topology;
+import io.zeebe.containers.ZeebeBrokerContainer;
+import io.zeebe.containers.ZeebeGatewayContainer;
+import io.zeebe.containers.ZeebePort;
+import io.zeebe.containers.ZeebeTopologyWaitStrategy;
+import io.zeebe.test.util.asserts.TopologyAssert;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+
+public class Ipv6IntegrationTest {
+  private static final int CLUSTER_SIZE = 3;
+  private static final int PARTITION_COUNT = 1;
+  private static final int REPLICATION_FACTOR = 3;
+  private static final String ZEEBE_IMAGE_VERSION = "camunda/zeebe:current-test";
+  private static final String NETWORK_ALIAS = "test-ipv6";
+  private static final String BASE_PART_OF_SUBNET = "2081::aede:4844:fe00:";
+  private static final String SUBNET = BASE_PART_OF_SUBNET + "0/123";
+
+  @Rule
+  public final Network network =
+      Network.builder()
+          .createNetworkCmdModifier(
+              createNetworkCmd ->
+                  createNetworkCmd
+                      .withIpam(new Ipam().withConfig(new Config().withSubnet(SUBNET)))
+                      .withName(NETWORK_ALIAS))
+          .enableIpv6(true)
+          .build();
+
+  private List<ZeebeBrokerContainer> containers;
+  private List<String> initialContactPoints;
+  private ZeebeGatewayContainer gateway;
+
+  @Before
+  public void setup() {
+    initialContactPoints = new ArrayList<>();
+    containers =
+        IntStream.range(0, CLUSTER_SIZE)
+            .mapToObj(i -> new ZeebeBrokerContainer(ZEEBE_IMAGE_VERSION))
+            .collect(Collectors.toList());
+
+    gateway = new ZeebeGatewayContainer(ZEEBE_IMAGE_VERSION);
+    IntStream.range(0, CLUSTER_SIZE).forEach(i -> configureBrokerContainer(i, containers));
+    configureGatewayContainer(gateway, initialContactPoints.get(0));
+  }
+
+  @After
+  public void tearDown() {
+    containers.parallelStream().forEach(GenericContainer::stop);
+  }
+
+  @Test
+  public void shouldCommunicateOverIpv6() {
+    // given
+    containers.parallelStream().forEach(GenericContainer::start);
+    gateway.start();
+
+    // when
+    final ZeebeClientBuilder zeebeClientBuilder =
+        ZeebeClient.newClientBuilder()
+            .usePlaintext()
+            .gatewayAddress(gateway.getExternalGatewayAddress());
+    final Topology topology;
```

Comment:
Just a nit pick, since the topology is only used in the try catch, you can just declare there.

Reply:
Done